oracle-plugin/src/e2e-test/java/io.cdap.plugin/OracleClient.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.TimeZone;
+
+/**
+ *  Oracle client.
+ */
+public class OracleClient {
+
+  private static Connection getOracleConnection() throws SQLException, ClassNotFoundException {
+    TimeZone timezone = TimeZone.getTimeZone("Asia/Kolkata");
+    TimeZone.setDefault(timezone);
+    Class.forName("oracle.jdbc.driver.OracleDriver");
+    String host = PluginPropertyUtils.pluginProp("ORACLE_HOST");
+    String username = PluginPropertyUtils.pluginProp("ORACLE_USERNAME");
+    String password = PluginPropertyUtils.pluginProp("ORACLE_PASSWORD");
+    int port = Integer.parseInt(PluginPropertyUtils.pluginProp("ORACLE_PORT"));
+    String database = PluginPropertyUtils.pluginProp("database");
+    return DriverManager.getConnection("jdbc:oracle:thin:@//" + host + ":" + port + "/" + database, username, password);
+  }
+
+  public static int countRecord(String table, String schema) throws SQLException, ClassNotFoundException {
+//    String countQuery = "SELECT COUNT(*) as total FROM " + schema + "." + table;
+    String countQuery = PluginPropertyUtils.pluginProp("selectQuery");
```

Comment:
why using select query for counting records?

Reply:
Changed it to count query